rss_reader/rss_reader.py

```diff
@@ -0,0 +1,112 @@
+"""Main module of the program"""
+
+import rss_reader.args_parser as args_parser
+import rss_reader.rss_parser as rss_parser
+import rss_reader.other as other
+import rss_reader.json_converter as json_converter
+import rss_reader.logs as logs
+import rss_reader.news as news
```

Comment:
Same here. It is better to use `from` in this case

Reply:
fixed